### DISCUSSION TITLE
help - how to list all available coc actions

### DISCUSSION THREAD

**Discussion by yatesco** [neoclide/coc.nvim#3430] [Q&A] [answered]
Hi there - how do I find a list of all available coc actions?

(I actually wanted to see if there was an action for indenting the entire buffer and I realised I don't actually know how to find things here.)

Thanks!

**Comment by xiyaowong** [comment, marked as answer]
https://github.com/neoclide/coc.nvim/blob/03c9add7cd867a013102dcb45fb4e75304d227d7/doc/coc.txt#L2311

**Reply by yatesco** [reply]
thanks - I'm kicking myself for not thinking of that ;-)

**Comment by chemzqm** [comment]
Consider use `'indentexpr'` or similar vim option for indent which should be faster and more flexiable than language server.
